spec/models/item_spec.rb

```diff
@@ -45,11 +45,11 @@
     expect(item).to respond_to(:categories)
   end
 
-  xit 'returns an empty item array' do
+  it 'returns an empty orders array' do
     expect(item.orders).to eq([])
   end
 
-  xit 'responds to orders' do
+  it 'responds to orders' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.